src/components/GitHubCards.js

```diff
@@ -1,64 +1,40 @@
 import React, { Component } from "react";
 import Axios from "axios";
-import baseURL from "../api/GitHub";
 import RepoCard from "./RepoCard";
 import { Grid } from "@material-ui/core";
+import {USER_NAME} from '../api/Github';
 
 class GitHubCards extends Component {
   Title = [];
   state = {
     repo: [],
     language: [],
   };
-  compare = (a, b) => {
-    if (a.stargazers_count < b.stargazers_count) {
-      return -1;
-    }
-    if (a.stargazers_count > b.stargazers_count) {
-      return 1;
-    }
-    return 0;
-  };
 
   async componentDidMount() {
     const api_key = process.env.REACT_APP_API_KEY;
 
-    let repo = [
-      baseURL(`covid-19-tracker`),
-      baseURL(`material-ui-medium-blog`),
-      baseURL(`github-readme-medium-card`),
-      baseURL(`react-youtube-search-clone`),
-      baseURL(`tamil-song-corpus`),
-      baseURL(`Nozama_Warriors`),
-      baseURL(`Nursery_Management`),
-      baseURL(`React-Medium-Blog`),
-    ];
+    let repos = await Axios.get(`https://api.github.com/users/${USER_NAME}/repos`);
```

Comment:
It is fetching all repositories. The aim of this project is to fetch specific repos. If user have 50 or 60 repositories it is not worth. Could you please restructure it

Reply:
user likes to only show his popular repositories on their website. Thank you for you PR. Nice work